src/styles/blocks/stars.scss

```
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+
+    background-image: url(/images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+}
+
+.card__stars_stars-4 :last-child,
```

```suggestion
.stars--4 .stars__star:nth-child(-n + 1),
```
maybe try a selector like this
and add a selector for 5 stars)